core-web/libs/sdk/client/src/lib/client/page/page-api.ts

```diff
@@ -263,7 +266,12 @@ export class PageClient {
             throw error;
         }
 
-        return response.json().then((data) => data.entity);
+        return response.json().then<DotCMSPageAPIResponse>((data) => ({
+            page: data.entity,
+            params: {
+                ...normalizedParams
+            }
+        }));
```

Comment:
Can we include the params inside pageAsset? That might be less intrusive, I think. Also, devs are already using this

Reply:
Something like:

```suggestion
        return response.json().then<DotCMSPageAPIResponse>((data) => ({
            ...data.entity,
            params // UI params No backend params for the UVE
        }));
```